src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -356,5 +356,14 @@ public override string TemplateHint
                 return DisplayMetadata.TemplateHint;
             }
         }
+
+        /// <inheritdoc />
+        public override IReadOnlyList<object> ValidatorMetadata
+        {
+            get
+            {
+                return new ReadOnlyCollection<object>(ValidationMetadata.ValiatorMetadata);
```

Comment:
creating a new collection everytime for something that is already cached? should we consider caching this as well?